apps/charterafrica/src/components/EmbeddedDocumentViewer/EmbeddedDocumentViewer.js

```diff
@@ -0,0 +1,75 @@
+import { Section } from "@commons-ui/core";
+import { Box } from "@mui/material";
+import React from "react";
+
+import useDocument from "./useDocument";
+
+import { secondary, neutral } from "@/charterafrica/colors";
+import RichText from "@/charterafrica/components/RichText";
+
+const EmbeddedDocumentViewer = React.forwardRef(function EmbeddedDocumentViewer(
+  props,
+  ref
+) {
+  const { documentUrl, excerpt, showNotes, title, sx } = props;
+
+  const { data } = useDocument(documentUrl, {
+    notes: showNotes,
+  });
+  const { html } = data || {};
+  if (!html?.length) {
+    return null;
+  }
+  return (
+    <Box
+      sx={{
+        backgroundColor: secondary[50],
+        pb: 5,
+        ...sx,
+      }}
+      ref={ref}
+    >
+      <Section
+        sx={{
+          px: { xs: 7.5, sm: 0 },
+        }}
+      >
+        <RichText
+          elements={title}
+          variant="h2SemiBold"
+          color="neutral.dark"
```

Comment:
```suggestion
          color="neutral.dark"
          elements={title}
          variant="h2"
```

A regular `RichTypography` here will do. A title isn't going to be formatted i.e. underlined, italicized, etc.